homeassistant/components/wilight/__init__.py

```diff
@@ -10,7 +10,7 @@
 from .parent_device import WiLightParent
 
 # List the platforms that you want to support.
-PLATFORMS = ["light"]
+PLATFORMS = ["light", "fan"]
```

Comment:
```suggestion
PLATFORMS = ["fan", "light"]
```

Reply:
Before committing your suggestion had passed all tests. Now it failed.
What would be the reason?

Reply:
https://github.com/home-assistant/core/pull/39541/checks?check_run_id=1736754805#step:8:94

Unless I'm overlooking something obvious this looks like a glitch just not finding asynctest ... maybe try with an empty commit (e.g. just retrigger the build). `git add -A ; git commit --allow-empty -S -m 'Retrigger' ; git push origin add_wilight_fan` or something likewise